components/bitly/actions/expand-bitlink/expand-bitlink.mjs

```diff
@@ -0,0 +1,38 @@
+import { axios } from "@pipedream/platform";
+import { ConfigurationError } from "@pipedream/platform";
+
+export default {
+  key: "bitly-expand-bitlink",
+  name: "Expand a Bitlink",
+  description: "Retrieves information about Bitlink using ID.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    bitly: {
+      type: "app",
+      app: "bitly",
+    },
+    bitlink_id: {
+      type: "string",
+      description: "Bitlink ID",
+    },
+  },
+  async run({ $ }) {
+    const { bitlink_id } = this;
+    const data = { bitlink_id };
+
+    try {
+      return await axios($, {
```

Comment:
Normally we use the `app` as a repository for all API calls and we just refer to it here. Take a look on `blogger` components for better reference :)